# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/platforms/PDMFactory.h
# Commit: c177bcba1c07
# Full Hash: c177bcba1c076d041e3470c8058bb2b4adaa1123
# Author: Dan Glastonbury <dan.glastonbury@gmail.com>
# Date: 2020-10-21 09:51:28
# Regressor Bug: 1595994
# File Overlap Count: 1
# Description:
#   Bug 1595994 - P8: Use RemoteDecoderModule to communicate with GPU and RDD. r=mattwoodrow
#   
#   Depends on D54879
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D54880
# ==============================================================================

diff -r 57607f8b6d8e -r c177bcba1c07 dom/media/platforms/PDMFactory.h
--- a/dom/media/platforms/PDMFactory.h	Tue Oct 20 23:26:27 2020 +0000
+++ b/dom/media/platforms/PDMFactory.h	Tue Oct 20 23:26:30 2020 +0000
@@ -63,6 +63,7 @@
   bool CreateAndStartupPDM(ARGS&&... aArgs) {
     return StartupPDM(DECODER_MODULE::Create(std::forward<ARGS>(aArgs)...));
   }
+
   // Startup the provided PDM and add it to our list if successful.
   bool StartupPDM(already_AddRefed<PlatformDecoderModule> aPDM,
                   bool aInsertAtBeginning = false);
